chore: add Ubuntu 22.04 Jammy Jellyfish

phil-davis opened the pull request:
We have PHP 7.4 available on Ubuntu 20.04.
This PR also makes it available on Ubuntu 22.04 so that we have the option to use that when we need to.

Note: I suspect that this will install PHP 8.1 on Ubuntu 22.04 by default, and I will have to make some changes to explicitly install PHP 7.4, similar to what is done for https://github.com/owncloud-ci/php

phil-davis commented:
https://drone.owncloud.com/owncloud-docker/php/548/2/2
```
5.316 E: Unable to locate package php-apcu-bc
5.316 E: Unable to locate package libx265-179
```
These errors are probably because there is PHP8.x, and actually I need to explicitly select PHP 7.4.

phil-davis commented:
https://ubuntu.pkgs.org/22.04/ubuntu-universe-amd64/libx265-199_3.5-2_amd64.deb.html

`libx265` seems to be at revision 199 for Ubuntu 22.04

phil-davis commented:
```
3.788 E: Unable to locate package php7.4-pear
3.788 E: Couldn't find any package by glob 'php7.4-pear'
3.788 E: Couldn't find any package by regex 'php7.4-pear'
```

https://www.freexian.com/lts/php/pecl-extensions/ does not have `pear` in the list.

I suppose that we don't need this anyway. I will drop it and see how CI goes.

phil-davis commented:
Now we get:
https://drone.owncloud.com/owncloud-docker/php/556/2/2
```
/bin/sh: 1: pecl: not found
```
I need to dig around to find out what is going on...

DeepDiver1975 commented:
> pecl install imagick

Let's use Ubuntu's imagick and do not build on our own for the time being - hopefully it is up to date enough

phil-davis commented:
I squashed, rebased and pushed.

After reviewing and merging, then I can do something in `owncloud-docker/base` that uses this Ubuntu 22.04, and provides an `owncloud-docker/base` thta has Ubuntu 22.04 with Freexian PHP 7.4.

and then make similar in `owncloud-docker/server` PR that demonstrates that basic tests are passing.

Then we can decide to switch `latest` to the Ubuntu 22.04 images.
